lib/util.ts

```diff
@@ -40,14 +40,16 @@ export function hashFileOrDirectory(fileOrDir: string): string {
   return hash.digest('base64');
 }
 
-export function renderEnvironmentVariables(env?: { [key: string]: string }, type?: cbuild.BuildEnvironmentVariableType) {
+export function renderEnvironmentVariables(env?: { [key: string]: string | undefined }, type?: cbuild.BuildEnvironmentVariableType) {
   if (!env) {
     return undefined;
   }
 
   const out: { [key: string]: cbuild.BuildEnvironmentVariable } = { };
   for (const [key, value] of Object.entries(env)) {
-    out[key] = { value, type };
+    if (value) {
```

Comment:
Maybe intentional - but are you certain you want to treat `""` as `undefined` here? Because both are false-y (this may effectively be the same thing though - not sure how CodeBuild behaves with empty string values).

Reply:
Mmm.. good point. Let me fix.

Reply:
Fixed